src/Compute/v2/Models/Server.php

```diff
@@ -230,6 +230,59 @@ public function revertResize()
     }
 
     /**
+     * Gets a VNC console for a server.
+     *
+     * @param  string $type The type of VNC console: novnc|xvpvnc.
+     *                      Defaults to novnc.
+     *
+     * @return mixed
+     */
+    public function getVncConsole($type = Enum::CONSOLE_NOVNC)
```

Comment:
Why is `$type` an arg here? Surely it'll always be `novnc` for this operation? I like removing all unnecessary expectations for users.